setup.py

```diff
@@ -9,10 +9,14 @@
     long_description_content_type="text/markdown",
     url="https://github.com/centrefornetzero/smart-building-rating-calculator",
     package_dir={"": "src"},
+    packages=find_packages(where="src"),
+    install_requires=[
+        "pandas",
```

Comment:
Can we be more specific with minimum pandas version + list all important versions, .e.g

```python

install_requires = (
    "pandas>=2.XX",
    "numpy>=1.25.0",
)

setup(
    ....
    install_requires=install_requires,
    ....
```

Reply:
Examples:
https://github.com/OpenSynth-energy/OpenSynth/blob/develop/setup.py

Reply:
How tight do we want to be with the bounds here? I think the only dep required for package is `pandas`. 

Currently in the env I have `pandas=2.2.2`, so the change here would just be be `install_requires = ["pandas>=2.2.2"]`?

Reply:
Yeah. If possible we want the bounds to be as wide as possible, i.e. also work with old versions of pandas. That way we can ensure that our software can be used in as many environments as possible to avoid the issue of people wanting to use our software, but they can't install because we have very tight bounds.

I would consider:
- Specifically with pandas, which features are we using, and when did these features appear in Pandas
- Safest option however is to do >=2.2.2 because we know for sure that's the case
- And in which case it just forces end users to upgrade their pandas to the latest version (which is something they should probably do anyway)

Reply:
The only place we're using pandas in the package is to use the `pd.cut` function, this was last changed in this PR to pandas - https://github.com/pandas-dev/pandas/pull/56101, which was part of [release `2.2.0`](https://pandas.pydata.org/pandas-docs/version/2.2.0/whatsnew/v2.2.0.html#other), so we can lower bound to `2.2` for sure. Can probably go lower, or potentially even remove the dependency on pandas (seems like a large dependency for just the `cut` function).

For this PR I think it's okay to bound to `"pandas>=2.2"`, and then potentially have a follow up PR to a) either remove the dependency or loosen this bound if there's a need for it. 

Does that sound okay?